lib/submission/submitter-payload.js

```diff
@@ -37,6 +26,22 @@ const generateEmail = (recipientType, from, subject, url, submissionId, addAttac
   return email
 }
 
+const emailTypes = {
+  USER: 'user',
+  TEAM: 'team'
+}
+
+const composeEmailBody = (userData, language, recipientType) => {
+  let body
+  if (recipientType === emailTypes.TEAM) {
```

Comment:
Are there only two types, `TEAM` and `USER`?

(Otherwise At 42 `body` will be undefined)

Reply:
Are you suggesting to add a guard? I could throw an error if invalid input is given.

Reply:
Relatedly, I suggest a switch statement:
```
switch (recipientType) { 
  case emailTypes.TEAM:
    body = getInstanceProperty('service', 'emailTemplateTeam') || 'Please find an application attached'
    break
  case emailTypes.USER:
    body = getInstanceProperty('service', 'emailTemplateUser') || 'A copy of your application is attached'
    break
  default: 
    body = 'A copy of the application is attached'
    break  
}
```